src/applications/ivc-champva/10-7959f-1/config/form.js

```diff
@@ -36,55 +32,71 @@ const formConfig = {
   trackingPrefix: '10-7959f-1-FMP-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  v3SegmentedProgressBar: true,
+  preSubmitInfo: {
+    statementOfTruth: {
+      body:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      messageAriaDescribedby:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      fullNamePath: formData => getNameKeyForSignature(formData),
```

Comment:
I believe this custom name path function can be left out if the signature doesn't need to match any text values elsewhere in the form (see other comment).

Reply:
Example without the `fullNamePath` property

```
preSubmitInfo: {
    statementOfTruth: {
      body:
        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
      messageAriaDescribedby:
        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
    }
}
```